app/src/main/kotlin/com/mapzen/erasermap/presenter/RoutePresenter.kt

```diff
@@ -6,13 +6,17 @@ import com.mapzen.valhalla.Instruction
 import com.mapzen.valhalla.Route
 
 public interface RoutePresenter {
+    companion object {
+        public val GESTURE_MIN_DELTA = 0.0001f
```

Comment:
Is certain hardware able to track sub-pixel motion gestures? I hadn't heard of that. Just wondering if it needs to be that fine or not.